bears/shell/ShellCheckBear.py

```diff
@@ -20,8 +20,14 @@ class ShellCheckBear:
     CAN_DETECT = {'Syntax', 'Security', 'Undefined Element', 'Unused Code'}
 
     @staticmethod
-    def create_arguments(filename, file, config_file, shell: str='sh'):
+    def create_arguments(filename, file, config_file, shell: str='sh',
+                         shellcheck_ignore: list=None):
         """
         :param shell: Target shell being used.
+        :param shellcheck_ignore: List of linting rules that should be ignored.
```

Comment:
I think i'll pass this one because coala mainly use term `_ignore`  rather than `exclude`, see https://github.com/coala/coala-bears/pull/1396#pullrequestreview-24900209